frontend/src/SettingsView/BillingView/constants/stripe.js

```diff
@@ -1,6 +1,6 @@
 const STRIPE_PUBLISHABLE =
 	process.env.NODE_ENV === 'production'
-		? 'pk_live_MY_PUBLISHABLE_KEY'
+		? 'pk_live_GVMtxntEIujtyI7ZOBA45dQG'
```

Comment:
You'll want to use the test key for the duration of the app development phase, otherwise you risk charges on the Stripe account for whoever signed up for it. Particularly when someone like Brian is testing your app as deployed in this case.